src/test/kotlin/sirius/kernel/commons/StringsTest.kt

```diff
@@ -332,4 +332,90 @@ class StringsTest {
+        assertEquals("…[truncated]", Strings.truncateMiddle("ABCDEFG", -10, 0))
+        assertEquals("…[truncated]", Strings.truncateMiddle("ABCDEFG", -10, -10))
+        // Szenario 8: input present and length is greater than input.length() and charactersToPreserveAtTheEnd is greater than length
+        assertEquals("ABCDEFG", Strings.truncateMiddle("ABCDEFG", 10, 10000))
+        assertEquals("ABCDEFG", Strings.truncateMiddle("ABCDEFG", 10, 11))
+        // Szenario 9: input present and length is greater than input.length() and charactersToPreserveAtTheEnd is greater than input.length()
+        assertEquals("ABCDEFG", Strings.truncateMiddle("ABCDEFG", 10, 8))
+        // Szenario 10: input present and length is greater than input.length() and charactersToPreserveAtTheEnd is equal to input.length()
+        assertEquals("ABCDEFG", Strings.truncateMiddle("ABCDEFG", 10, 7))
+        // Szenario 11: input present and length is smaller than input.length() and charactersToPreserveAtTheEnd is greater than length and length + charactersToPreserveAtTheEnd is smaller than input.length()
+        // Note: the ellipsis after the truncated signal is only added if there are characters to preserve at the end
+        // Note: the length of the truncated signal is only taken into account when limiting the input to length if length is greater than 13, otherwise the input will be truncated to the provided length and the truncated signal is added afterward
+        assertEquals("ABCDEFG-…[truncated]…This is the important part", Strings.truncateMiddle("ABCDEFG-ABCDEFG-This is the important part", 8, 26))
+        assertEquals("ABCDEFG-".length, 8)
+        assertEquals("This is the important part".length, 26)
+        // Szenario 12: input present and length is smaller than input.length() and charactersToPreserveAtTheEnd is greater than length and length + charactersToPreserveAtTheEnd is greater than input.length()
+        // Note: in the following case it is assumed that the whole string should be preserved because charactersToPreserveAtTheEnd is greater than input.length()
+        assertEquals("ABCDEFG-ABCDEFG-This is the important part", Strings.truncateMiddle("ABCDEFG-ABCDEFG-This is the important part", 0, 260))
+        assertEquals("ABCDEFG-ABCDEFG-This is the important part", Strings.truncateMiddle("ABCDEFG-ABCDEFG-This is the important part", -10, 260))
+        // Szenario 13: input present and length is smaller than input.length() and charactersToPreserveAtTheEnd is equal to length and length + charactersToPreserveAtTheEnd is equal to input.length()
+        // Note: in the following case it is assumed that the whole string should be preserved because length + charactersToPreserveAtTheEnd is equal to input.length()
+        assertEquals("ABCDEFG-ABCDEFG-This is the important part", Strings.truncateMiddle("ABCDEFG-ABCDEFG-This is the important part", 21, 21))
+        assertEquals("ABCDEFG-ABCDEFG-This is the important part".length, 42)
```

Comment:
same as above

Reply:
What is the issue here?
This seems fine to me?

Reply:
it doesnt test anything